src/declaration.d

```diff
@@ -996,6 +996,8 @@ public:
     Expression edtor;               // if !=null, does the destruction of the variable
     IntRange* range;                // if !=null, the variable is known to be within the range
 
+    ubyte _isDataSeg;               // 0b00 unset, 0b01 true, 0b11 false
```

Comment:
Since the value is basically a 2bit bit-field I thought it appropriate to write it as binary literal.
Otherwise it could be difficult to see why the value 2 is not used;
0 // unset
1 // true
3 // false